### ISSUE TITLE
Drop index incorrect syntax

### ISSUE THREAD

**Issue by isgroup-srl** [mmatuson/SchemaSync#49]
There should not be an `ON` statement after `DROP INDEX` inside an `ALTER TABLE` query (http://dev.mysql.com/doc/refman/5.7/en/alter-table.html). This behavior is legit if the `DROP INDEX` is used as a standalone query (http://dev.mysql.com/doc/refman/5.7/en/drop-index.html).

```
ALTER TABLE `foo` ADD COLUMN `x` bigint(20) unsigned NOT NULL AFTER `id`, DROP INDEX `y` ON `foo` [..]
```

Should be:

```
ALTER TABLE `foo` ADD COLUMN `x` bigint(20) unsigned NOT NULL AFTER `id`, DROP INDEX `y` [..]
```

**Comment by cpd1992** [comment]
Can anybody repair this issue？ Thanks a lot

**Comment by jacoryjin** [comment]
schemaobject->index.py->class IndexSchema(object)->drop,

if self.name == 'PRIMARY':
return "DROP PRIMARY KEY"
else:
return "DROP INDEX `%s` ON `%s`" % (self.name, self.parent.name)

changed to：

if self.name == 'PRIMARY':
return "DROP PRIMARY KEY"
else:
return "DROP INDEX `%s``" % (self.name)

**Comment by mmatuson** [comment]
@jacoryjin if you can submit a PR, I'll merge it

**Comment by stephenreay** [comment]
This should be fixed in mmatuson/SchemaObject#13

**Comment by stephenreay** [comment]
@mmatuson any chance of getting the referenced PR above merged in (as it should fix this issue)?
